src/compiler/core.ts

```diff
@@ -941,7 +945,7 @@ namespace ts {
      *  [^./]                   # matches everything up to the first . character (excluding directory seperators)
      *  (\\.(?!min\\.js$))?     # matches . characters but not if they are part of the .min.js file extension
      */
-    const singleAsteriskRegexFragmentFiles = "([^./]|(\\.(?!min\\.js$))?)*";
+    const singleAsteriskRegexFragmentFiles = "([^./]|(\\.(?!min\\.js$))?)*";

```

Comment:
It's line ending. This line ends with just an `LF` in master.

Which is strange, because we have a lint rule covering this which is not triggering.